site/src/pages/RoadmapPage/transfers/UncategorizedCreditsSection.tsx

```diff
@@ -32,7 +46,11 @@ const UncategorizedCreditsSection: FC = () => {
       </SectionDescription>
 
       {courses.map((course) => (
-        <UncategorizedMenuTile key={`${course.name}-${course.units}`} name={course.name} units={course.units} />
+        <UncategorizedMenuTile
+          key={`${course.name}-${course.units}`}
+          uncategorizedInfo={course}
+          unread={course.name ? unreadOther.includes(course.name) : false}
```

Comment:
Looks like this code is more clunky than it has to be because of typescript `undefined` complaints?

you could also do this:
```suggestion
          unread={unreadOther.includes(course.name ?? '')}
```